src/api/hooks/workers.ts

```diff
@@ -22,15 +22,21 @@ export const useWorker = (id: string, opts?: WorkerOpts) => {
 }
 
 type WorkersOpts = QueryHookOptions<GetWorkersQuery>
+export const storageWorkersVariables: GetWorkersQueryVariables = {
```

Comment:
we could just create `storageWorkersVariablesWhere` and don't pass `... storageWorkersVariables` below. Do we plan to add more properties here in the future?

Reply:
True, but I don't think it hurts us in any way to do this and we can always add something later